Dockerfile

```diff
@@ -68,27 +66,39 @@ RUN apt-get update \
   && rm -rf /var/lib/apt/lists/*
 
 ARG SPARK_VERSION
-ARG SPARK_VARIANT
+ARG HADOOP_VERSION
 ARG SPARK_PY4J
 
+ENV HADOOP_HOME /opt/hadoop
+ENV PATH ${PATH}:${HADOOP_HOME}/bin:${HADOOP_HOME}/sbin
+ENV HADOOP_CONF_DIR ${HADOOP_HOME}/etc/hadoop
+ENV HDFS_CONF_DIR ${HADOOP_CONF_DIR}
+ENV YARN_CONF_DIR ${HADOOP_CONF_DIR}
+ENV HIVE_CONF_DIR ${HADOOP_CONF_DIR}
```

Comment:
No point specifying these directories, since the user will almost certainly override them

Reply:
Removed and updated README with some instructions.